src/03_modules.py

```diff
@@ -10,22 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+for arguments in sys.argv:
+    print(str(arguments))
```

Comment:
Those arguments come through as strings by default even if it looks like an int or float, so no need to cast them, but doesn't hurt anything to do so!